helm/draughtsman-chart/templates/rbac.yaml

```diff
@@ -35,6 +35,7 @@ rules:
       - clusterroles
     verbs:
       - bind
+      - create
```

Comment:
Are you sure we need new verb?

Reply:
draughtsman using helmclient, and helmclient have to create `ClusterRole` if cluster does not have a tiller yet. 
https://github.com/giantswarm/helmclient/blob/386916cdcc22ddd332022e045d26d0e345562116/ensure_tiller_installed.go#L126-L158

Reply:
Yes, Ji's testing on a cluster uncovered that.